uefi_test.go

```diff
@@ -43,11 +44,34 @@ func TestAppendUEFIFirmwareDeviceConfig(t *testing.T) {
 		t.Errorf("Expected non-empty qemuParams, found %s", c.qemuParams)
 	}
 
-	expected := "-drive if=pflash,format=raw,readonly,file=OVMF_CODE.fd -drive if=pflash,format=raw,file=OVMF_VARS.fd"
+	expected := "-drive if=pflash,format=raw,readonly=on,file=OVMF_CODE.fd -drive if=pflash,format=raw,file=OVMF_VARS.fd"
 	result := strings.Join(c.qemuParams, " ")
 	if expected != result {
 		t.Fatalf("Failed to append parameters\nexpected[%s]\n!=\nfound   [%s]", expected, result)
 	}
 }
-
+/* Commented out because success/failure depends on local filesystem
+func TestNewFirmwareDev(t *testing.T) {
+	secureBoot := true
+	udev, err := NewSystemUEFIFirmwareDevice(secureBoot)
+	if err != nil {
+		t.Fatalf("Failed to find secure firmware blobs: %s", err)
+	}
+	switch runtime.GOARCH{
+	case "amd64":
+		if PathExists(UbuntuSecVars) {
+			expected := "-drive if=pflash,format=raw,readonly=on,file=/usr/share/OVMF/OVMF_CODE.secboot.fd -drive if=pflash,format=raw,file=/usr/share/OVMF/OVMF_VARS.ms.fd"
+			testAppend(*udev, expected, t)
+		} else if PathExists(CentosSecVars){
+			expected := "-drive if=pflash,format=raw,readonly=on,file=/usr/share/OVMF/OVMF_CODE.secboot.fd -drive if=pflash,format=raw,file=/usr/share/OVMF/OVMF_VARS.secboot.fd"
+			testAppend(*udev, expected, t)
+		} else {
+			t.Fatalf("Failed to find secure firmware blobs")
+		}
+	case "arm64", "aarch64":
+		expected := "-drive if=pflash,format=raw,readonly=on,file=/usr/share/AAVMF/AAVMF_CODE.ms.fd -drive if=pflash,format=raw,file=/usr/share/AAVMF/AAVMF_VARS.ms.fd"
+		testAppend(*udev, expected, t)
```

Comment:
Here you'll want the same Vars selection logic for arm64 path; (if UbuntuVars, else Centos, else fail).